types/jszip/index.d.ts

```diff
@@ -37,6 +39,7 @@ interface InputByType {
 interface OutputByType {
     base64: string;
     text: string;
+    string: string;
```

Comment:
As you ticked tests option, can you add a case for this change? e.g. line like:
```ts
newJszip.file("stream.txt").async('string')....
```
Otherwise, LGTM!

Reply:
Good idea, added a test.